bin/plot_mosdepth_regions.r

```diff
@@ -95,31 +95,34 @@ for (sample in unique(dat$sample)) {
         plot <- ggplot(sample_dat,aes(x=region,y=coverage)) +
                 geom_bar(stat="identity", fill="#D55E00", width=0.6) +
                 theme_bw() +
-                theme(plot.title=element_text(size=10),
```

Comment:
To be fair I find more readable the old formatting, but let's 🌈 make its magic...

Reply:
Yep, I cheated a bit in the `.r` files because R isn't really fussed about indentation and I am not aware of a nice way to make it all nicely consistent and indented. But given these aren't template files we can change that later.